main.js

```diff
@@ -6,7 +6,13 @@
  * @typedef {import("./typings/metrics").TickingMetric} TickingMetric
  */
 
-require('isomorphic-fetch');
+// only include isomorphic-fetch if the application has explicitly disabled native fetch
+if (
+	process.allowedNodeEnvironmentFlags.has('--no-experimental-fetch') &&
+	process.execArgv.includes('--no-experimental-fetch')
```

Comment:
**nitpick**: not that we support said versions anymore, but `process.allowedNodeEnvironmentFlags.has('--no-experimental-fetch')` resolving to `false` implies that this is an older Node version without support for native fetch so we should fallback to `isomorphic-fetch` in that case

```suggestion
	!process.allowedNodeEnvironmentFlags.has('--no-experimental-fetch') ||
	process.execArgv.includes('--no-experimental-fetch')
```

Reply:
oh very good point